libnavigation-ui/src/main/java/com/mapbox/navigation/ui/voice/SpeechPlayerProvider.java

```diff
@@ -168,4 +216,66 @@ private void initializeAndroidSpeechPlayer(Context context, String language,
     androidSpeechPlayer = new AndroidSpeechPlayer(context, language, listener);
     speechPlayers.add(androidSpeechPlayer);
   }
+
+  /**
+   * The Builder of {@link SpeechPlayerProvider}.
+   */
+  public static class Builder {
+    // static field to support SpeechPlayerProvider deprecated constructor
+    private static final int DEFAULT_FOCUS_GAIN = AudioManager.AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK;
+
+    @NonNull
+    private final  Context context;
+    private final  String language;
+    private final boolean voiceLanguageSupported;
+    @NonNull
+    private final VoiceInstructionLoader voiceInstructionLoader;
+    private int focusGain = DEFAULT_FOCUS_GAIN;
+
+    /**
+     * A new instance of {@link Builder}.
+     *
+     * @param context                for the initialization of the speech players
+     * @param language               to be used
+     * @param voiceLanguageSupported true if <tt>voiceLanguage</tt> is not null, false otherwise
+     * @param voiceInstructionLoader voice instruction loader
+     */
+    public Builder(
+      @NonNull Context context,
+      String language,
+      boolean voiceLanguageSupported,
+      @NonNull VoiceInstructionLoader voiceInstructionLoader
+    ) {
+      this.context = context;
+      this.language = language;
+      this.voiceLanguageSupported = voiceLanguageSupported;
+      this.voiceInstructionLoader = voiceInstructionLoader;
+    }
+
+    /**
+     * Audio focus gain. One of {@link AudioManager#AUDIOFOCUS_GAIN_TRANSIENT},
+     * {@link AudioManager#AUDIOFOCUS_NONE}, and etc.
+     * <p>
+     * Default is {@link AudioManager#AUDIOFOCUS_GAIN_TRANSIENT_MAY_DUCK}
+     * @see AudioManager#requestAudioFocus(AudioManager.OnAudioFocusChangeListener, int, int)
+     * @see AudioManager#requestAudioFocus(AudioFocusRequest)
+     */
+    public Builder audioFocusGain(int focusGain) {
```

Comment:
In https://github.com/mapbox/mapbox-navigation-android/blob/ef29a985912779e4497e53d4ecb0b7c4e6f33330/libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/options/VoiceInstructionsPlayerOptions.kt#L74-L81 we're checking also if `focusGain` is a valid one.